From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Urlings <tom.urlings@gmail.com>
Date: Sun, 23 Nov 2025 07:42:00 +0000
Subject: Patching u-boot mvebu files include/configs/helios4.h

Re-applied build/patch/u-boot/legacy/u-boot-helios4/target_spi/helios4-reorder-boot-device.patch

Signed-off-by: Tom Urlings <tom.urlings@gmail.com>
---
 include/configs/helios4.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/helios4.h b/include/configs/helios4.h
index b0c23102c40..872b0c7717f 100644
--- a/include/configs/helios4.h
+++ b/include/configs/helios4.h
@@ -83,15 +83,15 @@
 /*
  * The SCSI buses are attempted in increasing bus order, there is no current
  * mechanism to alter the default bus priority order for booting.
  */
 #define BOOT_TARGET_DEVICES(func) \
-	BOOT_TARGET_DEVICES_MMC(func) \
 	BOOT_TARGET_DEVICES_USB(func) \
 	BOOT_TARGET_DEVICES_SCSI_BUS0(func) \
 	BOOT_TARGET_DEVICES_SCSI_BUS1(func) \
 	BOOT_TARGET_DEVICES_SCSI_BUS2(func) \
+	BOOT_TARGET_DEVICES_MMC(func) \
 	func(PXE, pxe, na) \
 	func(DHCP, dhcp, na)
 
 #define KERNEL_ADDR_R	__stringify(0x2080000)
 #define FDT_ADDR_R	__stringify(0x2040000)
-- 
Created with Armbian build tools https://github.com/armbian/build
